chapters/es/chapter2.md

```diff
@@ -336,7 +336,7 @@ pattern = [{"LOWER": "silicon"}, {"TEXT": " "}, {"LOWER": "valley"}]
 ```
 
 ```python
-doc = nlp("Can Silicon Valley workers rein in big tech from within?")
+doc = nlp("¿Pueden los trabajadores de Silicon Valley controlar las grandes empresas de tecnología desde dentro?")
```

Comment:
Could we find another example sentence here that's a bit shorter and doesn't require scrolling or a line break?

Reply:
```suggestion
doc = nlp("¿Por qué Silicon Valley necesita miles de astrofísicos?")
```